com.unity.render-pipelines.universal/Shaders/Utils/ScreenSpaceShadows.shader

```diff
@@ -63,7 +59,7 @@ Shader "Hidden/Universal Render Pipeline/ScreenSpaceShadows"
             deviceDepth = 2 * deviceDepth - 1; //NOTE: Currently must massage depth before computing CS position.
 
             float3 vpos = ComputeViewSpacePosition(input.uv.zw, deviceDepth, unity_CameraInvProjection);
-            float3 wpos = mul(unity_CameraToWorld, float4(vpos, 1)).xyz;
+            float3 wpos = mul(unity_CameraToWorld, float4(vpos.x, vpos.y, -vpos.z, 1)).xyz;
```

Comment:
I am also not sure why this is needed.

Reply:
Before binding it(https://github.com/Unity-Technologies/Graphics/blob/master/com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs#L87)
don't need to reverse z-value. I compared it with the previous code, there is no those binding vars.
I don't know which one is correct in Unity(left or right handed coordinates)

Reply:
Hm, I wonder if this in an issue in the ComputeViewSpacePosition function? The z-reverse is platform specific and it should be handled in this case by our unity_CameraInvProjection. 

```
float3 ComputeViewSpacePosition(float2 positionNDC, float deviceDepth, float4x4 invProjMatrix)
{
    float4 positionCS = ComputeClipSpacePosition(positionNDC, deviceDepth);
    float4 positionVS = mul(invProjMatrix, positionCS);
    // The view space uses a right-handed coordinate system.
    positionVS.z = -positionVS.z;
    return positionVS.xyz / positionVS.w;
}
```

Quick way to test: switch to OpenGL. :)

Reply:
I tried to test. it was the same result. I need to analyze how to handle it :) thanks.